### ISSUE TITLE
Ring Two-Factor Authentication

### ISSUE THREAD

**Issue by markcanup** [dcmeglio/hubitat-ring#3]
Hi,

I have two-factor authentication (2FA) enabled for my Ring account. I've loaded the app and the drivers into Hubitat, and when I went to authenticate in the app, I was prompted for my credentials. I entered them and received a text from Ring with my verification code, but the app in Hubitat just said I had entered invalid credentials. It does not seem that 2FA is supported by this integration.

I would be willing to temporarily disable 2FA to set this up and then re-enable 2FA, but I don't want to have to disable 2FA permanently for this integration to work.

Thanks...

**Comment by jarvis2017** [comment]
Is there any plan to support 2FA??

**Comment by dcmeglio** [comment]
No. I haven't maintained this in a long time. There are much better Hubitat Ring integrations out there and I don't think keeping this one going really helps anything
